src/core/lib/security/credentials/ssl/ssl_credentials.h

```diff
@@ -69,7 +69,16 @@ class grpc_ssl_credentials : public grpc_channel_credentials {
                     grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
                     const grpc_ssl_verify_peer_options* verify_options);
 
+  grpc_security_status InitializeClientHandshakerFactory(
```

Comment:
nit: Can we mark this as `static`?

Reply:
Friendly ping :)

Reply:
See the thread above this - this initializes the `client_handshaker_factory_` on this class instance of credentials. But if there's a session cache we use it to make a new factory.
I don't think we want it to be static - it uses and modifies the instance `client_handshaker_factory_`